### ISSUE TITLE
Document all env variables

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/radical.pilot#1058]
including PTY timeout, log targets, profile, pilot debug etc.

**Comment by andre-merzky** [comment]
```
R:

  RADICAL_VERBOSE
  RADICAL_LOG_TGT
  
  
  
RP: 
  
  RADICAL_PILOT_AGENT_CONFIG
  RADICAL_PILOT_AGENT_VERBOSE
  RADICAL_PILOT_DBURL
  RADICAL_PILOT_RECORD_SESSION
  
  RADICAL_PILOT_VERBOSE
  RADICAL_PILOT_LOG_TGT



SAGA:
  
  EC2_URL
  EC2_ACCESS_KEY
  EC2_SECRET_KEY
  X509_USER_PROXY
  
  SAGA_GLOBUS_ONLINE_LOCALHOST_ENDPOINT
  SAGA_ADAPTOR_PATH
  SAGA_PTY_SSH_COPYMODE
  SAGA_PTY_SSH_SHAREMODE
  SAGA_PTY_SSH_TIMEOUT
  SAGA_PTY_CONN_POOL_TTL
  SAGA_PTY_CONN_POOL_SIZE
  SAGA_PTY_CONN_POOL_WAIT
  
  RADICAL_SAGA_VERBOSE
  RADICAL_SAGA_LOG_TGT
  RADICAL_SAGA_PTY_VERBOSE
  RADICAL_SAGA_PTY_LOG_TGT
```

**Comment by andre-merzky** [comment]
closed in favor of #1578
